.gitignore

```diff
@@ -15,20 +15,20 @@ dist
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-coverage
 
 .DS_Store
-coverage
 sitemap.xml
 
 # cypress
 cypress/downloads
 cypress/screenshots
 cypress/videos
 
-codecov*
+codecov
```

Comment:
Do we need this `codecov`?

Reply:
We don't need it. Sorta. it's just so that the `codcov` binary doesn't show up as a change in git. I'll remove it to align with other apps.